docs/administration/high-volume-log-loss.adoc

```diff
@@ -0,0 +1,362 @@
+= High volume log loss
+
+This guide explains how to handle scenarios where high-volume logging can cause log loss in OpenShift clusters, and how to configure your cluster to minimize this risk.
+
+== The problem
+
+To summarize the logging process on a cluster node:
+
+- **Containers** write to log files under `/var/log`.
+- **Kubelet** _rotates_ log files to enforce a size limit per container.
+  Rotation permanently deletes old logs to free disk space.
+- **Log Forwarder** collects logs by reading from `/var/log` files and sending to a remote store (often Loki).
+  - *Collected logs* have been read by the forwarder. They may be sent or still sending, but they are the forwarder's responsibility.
+  - *Uncollected logs* have not yet been read by the forwarder.
+
+.Log loss
+IMPORTANT: If uncollected logs are deleted by Kubelet before the forwarder can read them, they are **lost** permanently.
+
+.Filling `/var/log` will destabilize the node.
+WARNING: Kubelet rotation controls the size of log files in `/var/log`. The forwarder is responsible for controlling any other files (e.g. buffers) that it stores here.
+
+Log loss depends on three factors:
+
+[horizontal]
+write rate:: The rate that logs are written to `/var/log` files (bytes/sec)
+send rate:: The rate that logs are successfully sent to the remote store (bytes/sec)
+storage capacity:: Storage capacity for logs in `/var/log`
+
+The logging system has three modes of operation:
+
+Normal operation (no log loss)::
+Send-rate exceeds write-rate. Logs are forwarded as quickly as they arrive.
+The set of uncollected logs is small and does not grow significantly.
+The forwarder has time to collect all logs before they are rotated.
+
+Overload (possible log loss)::
+Write-rate exceeds send-rate (on average) for an extended period of time.
+This could be caused by high write activity and/or slow sending.
+Uncollected logs accumulate as the forwarder falls behind.
+A brief overload (sometimes called a _burst_ or _spike_) can be handled without log loss if there is enough storage.
+An overload that continues long enough force rotation of uncollected logs **will** cause log loss.
+
+Recovery (no log loss, but not yet back to normal)::
+After an overload, there are more uncollected logs accumulated in storage than during normal operation.
+We must consider the time it takes to reduce this backlog and get back to normal, before the next overload.
+
+IMPORTANT: It is theoretically impossible to prevent log loss under all conditions.
+The goal is to analyze your expected average and peak conditions, then provision storage accordingly.
+
+=== Calculations and examples
+
+*Time to log loss during steady overload:*
+
+When write rate consistently exceeds send rate, logs accumulate until storage fills:
+
+```
+Time to log loss = Storage capacity ÷ (Write rate - Send rate)
+```
+
+*Example - Steady overload:*
+- Storage capacity: 500MB (kubelet default: 10MB × 5 files × 10 containers)
```

Comment:
I think you wanted this to be a bullet list but it does not render that way

Reply:
+!, missing a newline